process/sync/baseSync.go

```diff
@@ -846,7 +846,7 @@ func (boot *baseBootstrap) requestMiniBlocksByHashes(hashes [][]byte) {
 // that will be added. The block executor should decide by parsing the header block body type value
 // what kind of block body received.
 func (boot *baseBootstrap) getMiniBlocksRequestingIfMissing(hashes [][]byte) (block.MiniBlockSlice, error) {
-	miniBlocks, missingMiniBlocksHashes := boot.miniBlocksResolver.GetMiniBlocksFromPool(hashes)
+	miniBlocks, missingMiniBlocksHashes := boot.miniBlocksDataGetter.GetMiniBlocksFromPool(hashes)
```

Comment:
right now you have `miniBlocksDataGetter.GetMiniBlocksFromPool(hashes)` . Wouldn't it be better after refactoring ? when you will have:
`miniBlocksDataProvider.MiniBlocksFromPool(hashes)`

Reply:
I think we will stick for the time being to the old naming (with get) as we know that a get action is performed. We might refactor these if we decide to move the functionality from process/common.go into specialized components like this provider.

Reply:
Ok